cmd/account/cli.go

```diff
@@ -71,20 +71,29 @@ func (o *cliOptions) complete(cmd *cobra.Command) error {
 }
 
 func (o *cliOptions) run() error {
-	awsClient, err := o.k8sclusterresourcefactory.GetCloudProvider(o.verbose)
+	_, err := o.k8sclusterresourcefactory.GetCloudProvider(o.verbose)
 	if err != nil {
 		return err
 	}
 
-	credentials, err := awsprovider.GetAssumeRoleCredentials(awsClient, &o.k8sclusterresourcefactory.Awscloudfactory.ConsoleDuration,
-		o.k8sclusterresourcefactory.Awscloudfactory.CallerIdentity.UserId,
-		aws.String(fmt.Sprintf("arn:aws:iam::%s:role/%s",
-			o.k8sclusterresourcefactory.AccountID,
-			o.k8sclusterresourcefactory.Awscloudfactory.RoleName)))
-	if err != nil {
-		return err
+	creds := o.k8sclusterresourcefactory.Awscloudfactory.Credentials
+
+	if o.output == "default" {
+		fmt.Fprintf(o.IOStreams.Out, "Temporary AWS Credentials:\n%s\n", creds)
+	}
+
+	if o.output == "json" {
```

Comment:
json and default have the same output minus a few words of preamble. Can we consolidate these?

Reply:
json == default without the pre-text. The pre-text was there before I started. Keeping that as `default` to maintain compatibility.

Personally, I wanted the `env` output for testing, the `json` is just an extra for almost zero cost.

I vote to leave it as is.

Reply:
Keeping default to maintain compatibility with what?

Reply:
OSD-5676 created to address the global output and cleaning it up across the entire application.